### PR TITLE
Add clearAllRadialItems

### PR DESCRIPTION
**Pull request by Mkeefeus**
I used this for our conversion from qb-radialmenu to clear the menu then only add a "call for help" option or similar. Figured it could be useful in other niche situations

### CODE REVIEW

File: resource/interface/client/radial.lua [hunk 1]
```diff
@@ -182,6 +182,15 @@ function lib.removeRadialItem(id)
     refreshRadial(id)
 end
 
+---Removes all items from the global radial menu.
+function lib.clearAllRadialItems()
```
Comment (LukeWasTakenn) [inline resource/interface/client/radial.lua:186]: I think `clearRadialItems` would be a bit better than `clearAllRadialItems`

File: resource/interface/client/radial.lua [hunk 2]
```diff
@@ -182,6 +182,15 @@ function lib.removeRadialItem(id)
...
+    if isOpen then
+        lib.hideRadial()
+    end
```
Comment (LukeWasTakenn) [inline resource/interface/client/radial.lua:189]: Why not do a check for `not currentRadial` as well and update the menu instead of closing it?

**Review by LukeWasTakenn — CHANGES_REQUESTED** [review]
Could you also make a PR to the docs for this function?

**Comment by Mkeefeus** [comment]
Requested updates made and the PR for docs is open

**Review by LukeWasTakenn — APPROVED** [review]

**Pull request merged**